.eslintrc

```diff
@@ -3,9 +3,9 @@
       "commonjs": true,
       "es6": true,
       "node": true,
-      "browser": true
+      "jest": true
```

Comment:
Do we need `node`? We should probably keep `browser`.

Reply:
Also, is the serviceWorker file properly linted without the `serviceWorker` env here?